### PR TITLE
Update of Strip hit efficiency code output

### PR DESCRIPTION
**Pull request by jlagram**
#### PR description:

Small changes to SiStripHitEffFromCalibTree.cc:
- add the possibility as an option to output the efficiencies for each module, not only layer average
  -> additional tree in the output root file
- add PU/lumi distributions including the effect of the event selection based on track multiplicity
  -> additional histos in the output root file 

#### PR validation:

Ran the code and checked output content.

### CODE REVIEW

File: CalibTracker/SiStripHitEfficiency/plugins/SiStripHitEffFromCalibTree.cc [hunk 1]
```diff
@@ -1004,6 +1034,22 @@ void SiStripHitEffFromCalibTree::makeHotColdMaps() {
 
 void SiStripHitEffFromCalibTree::makeTKMap(bool autoTagging = false) {
   LOGPRINT << "Entering TKMap generation!\n";
+
+  TTree* tree;
```
Comment (tvami) [inline CalibTracker/SiStripHitEfficiency/plugins/SiStripHitEffFromCalibTree.cc:1038]: so CLANG wants this to be initialized
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-bf7a0b/32946/new-build-warnings.log

**Comment by tvami** [comment]
hi @jlagram after fixing the CLANG warning, please squash the commits and modify the PR title to be a bit more descriptive. Thanks

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by tvami** [comment]
> hi @jlagram after fixing the CLANG warning, please squash the commits and modify the PR title to be a bit more descriptive. Thanks

Thanks for fixing the warning and the PR title change. Please dont forget about the squash either

**Comment by tvami** [comment]
+alca
 * last push is just a squash

**Comment by perrotta** [comment]
+1

**Pull request merged**